projects/plugins/jetpack/extensions/shared/components/audio-player/index.jsx

```diff
@@ -109,7 +124,7 @@ function AudioPlayer( {
 			onError && audio.removeEventListener( 'error', onError );
 			onMetadataLoaded && audio.removeEventListener( 'loadedmetadata', onMetadataLoaded );
 		};
-	}, [ audioRef, onPlay, onPause, onError, onJumpBack, onSkipForward, onMetadataLoaded ] );
+	}, [ onPlay, onPause, onError, onJumpBack, onSkipForward, onMetadataLoaded ] );
```

Comment:
Why is `audioRef` no longer needed in the `useEffect` dependency array?

Reply:
I think it wasn't needed previously - `audioRef.current` might change, but not `audioRef` itself. The clean up function is still run as the component unmounts ( like when we click on Replace to change the feed ), but otherwise it only runs as the AudioPlayer component is mounted.